.pre-commit-config.yaml

```diff
@@ -18,6 +18,14 @@ repos:
     hooks:
       - id: black-jupyter
 
+  - repo: https://github.com/pycqa/isort
+    rev: 5.12.0
+    hooks:
+    - id: isort
+      name: isort (python)
+      args:
+      - --profile=black
```

Comment:
Just a note that [`isort` is already being run through `ruff`](https://beta.ruff.rs/docs/faq/#how-does-ruffs-import-sorting-compare-to-isort), just 100x faster. So this is not necessary and might clash with `ruff`.

Reply:
Oh, I've never used rust. I'll remove it then. thanks for pointing it out!